ddtrace/contrib/botocore/patch.py

```diff
@@ -61,28 +61,41 @@ class TraceInjectionDecodingError(Exception):
     pass
 
 
-def inject_trace_data_to_message_attributes(trace_data, entry):
-    # type: (Dict[str, str], Dict[str, Any]) -> None
+def inject_trace_data_to_message_attributes(trace_data, entry, endpoint):
+    # type: (Dict[str, str], Dict[str, Any], endpoint) -> None
```

Comment:
```suggestion
    # type: (Dict[str, str], Dict[str, Any], str) -> None
```

?

Reply:
or... if it _has_ to be `sns` or `sqs` we can do:

```suggestion
    # type: (Dict[str, str], Dict[str, Any], Union[Literal["sqs"], Literal["sns"]]) -> None
```

Reply:
The other type hints in this doc need to be updated as well, "endpoint" isn't a type, so should be failing mypy/type checking.